src/main/java/ro/msg/learning/shop/repository/StockRepository.java

```diff
@@ -0,0 +1,11 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.entity.StockEntity;
+
+import java.util.UUID;
+
+@Repository
+public interface StockRepository extends JpaRepository<StockEntity, UUID>
```

Comment:
Should be public interface public interface StockRepository extends JpaRepository<StockEntity, StockID>  (StockID is the type of the id in the StockEntity)